be/src/controllers/rest.router.ts

```diff
@@ -129,9 +129,32 @@ router.delete('/vocab/:id', (req: Request, res: Response, next: NextFunction) =>
         .catch(next);
 });
 
-router.get('/vocab/:id/entities', (req: Request, res: Response, next: NextFunction) => {
+router.get('/vocab/:vId/entities', (req: Request, res: Response, next: NextFunction) => {
```

Comment:
Not sure if we need `list Entities` in this PR

Reply:
The ticket just states reading entities, that is why I included GET entitiy and GET entities, but the latter without its search functionality, which will be done in an extra branch